<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XPlayerClass</class>
 <widget class="QMainWindow" name="XPlayerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>820</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XPlayer</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QOpenGLWidget" name="m_wndScreen">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>750</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QSlider" name="m_sldProgress">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>750</y>
      <width>1280</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal { 
	border: 1px solid gray;
	height: 2px; 
	left: 4px; 
	right: 4px; 
} 
QSlider::handle:horizontal { 
	border: 1px solid gray; 
	background:white;
	border-radius: 5px;
	width: 10px; 
	height: 10px; 
	margin: -5px; 
} 
QSlider::sub-page:horizontal{ 
	background: rgb(0, 255, 0); 
}</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="m_btnCtrl">
    <property name="geometry">
     <rect>
      <x>615</x>
      <y>770</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="XPlayer.qrc">
      <normaloff>:/XPlayer/res/pause.ico</normaloff>
      <normalon>:/XPlayer/res/play.ico</normalon>:/XPlayer/res/pause.ico</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="m_btnStop">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>770</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>res/stop.ico</normaloff>res/stop.ico</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="m_btnBackward">
    <property name="geometry">
     <rect>
      <x>565</x>
      <y>770</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="XPlayer.qrc">
      <normaloff>:/XPlayer/res/backward.ico</normaloff>:/XPlayer/res/backward.ico</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="m_btnForward">
    <property name="geometry">
     <rect>
      <x>665</x>
      <y>770</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="XPlayer.qrc">
      <normaloff>:/XPlayer/res/forward.ico</normaloff>:/XPlayer/res/forward.ico</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="m_btnLast">
    <property name="geometry">
     <rect>
      <x>515</x>
      <y>770</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="XPlayer.qrc">
      <normaloff>:/XPlayer/res/last.ico</normaloff>:/XPlayer/res/last.ico</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="m_btnNext">
    <property name="geometry">
     <rect>
      <x>715</x>
      <y>770</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="XPlayer.qrc">
      <normaloff>:/XPlayer/res/next.ico</normaloff>:/XPlayer/res/next.ico</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="XPlayer.qrc"/>
 </resources>
 <connections/>
</ui>
